gremlin-test/src/main/resources/org/apache/tinkerpop/gremlin/test/features/map/Order.feature

```diff
@@ -34,24 +34,6 @@ Feature: Step - order()
       | ripple |
       | vadas  |
 
-  Scenario: g_V_name_order_byXa1_b1X_byXb2_a2X
```

Comment:
Were these scenarios removed because of an issue with the lambda? Was there something about the changes in this PR which causes them to fail where they previously passed?

Reply:
removed for lambdas. given https://issues.apache.org/jira/browse/TINKERPOP-3136 it was more trouble to figure out the problem than to just get a head start on that issue. these tests already exist in embedded.